tests/queries/models.py

```diff
@@ -737,3 +737,57 @@ class RelatedIndividual(models.Model):
 
     class Meta:
         db_table = 'RelatedIndividual'
+
+
+class SchemaQualified(models.ModelTable):
+    def __init__(self, schema, table):
+        self.schema = schema
+        self.table = table
+
+    def as_sql(self, compiler, connection):
+        # Note that it would be easy to change the schema using thread locals
+        # if wanted.
```

Comment:
Yes, just like the dynamic query class below uses a subquery when db_time is set, you could use different schemata using either threadlocals or query.context.

Notably we wont have official support for this for some time. In practice this means you'll have to do migrations manually.